core/src/main/java/org/microshed/testing/jwt/JwtConfigExtension.java

```diff
@@ -0,0 +1,99 @@
+package org.microshed.testing.jwt;
+
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionConfigurationException;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.microshed.testing.internal.InternalLogger;
+import org.microshed.testing.jupiter.MicroShedTestExtension;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+public class JwtConfigExtension implements BeforeEachCallback, AfterEachCallback {
```

Comment:
Since we expect this annotation to be placed on methods and not classes should we swap this for BeforeTestExecutionCallback?
```suggestion
public class JwtConfigExtension implements BeforeTestExecutionCallback, AfterTestExecutionCallback {
```